tests/layerstack.stanza

```diff
@@ -154,3 +150,78 @@ deftest(layerstack) test-with-soldermask:
   #EXPECT(sm2? is-not None)
   #EXPECT(core-1? is-not None)
 
+;============================================================================
+;<note>
+My Crazy Stack - each layer is different for testing
+ 0 - copper-3oz
+ 1 - core-FR4-2
+ 2 - copper-2oz
+ 3 - prepreg-FR4
+ 4 - copper-1oz
+ 5 - core-FR4-1
+ 6 - copper-1oz
+ 7 - prepreg-FR4
+ 8 - copper-2oz
+ 9 - core-FR4-2
+ 10 - copper-3oz
+;<note>
+val test-stack = LayerStack(name = "My Crazy Stack")
+val copper-1mm = Copper(1.0, name = "cu1")
```

Comment:
We should use :

`1mm` -> `35um` 
`2mm` -> `70um` 
`3mm` -> `105um` 

For more realistic values.